benchmark/flexsn/benchmark_vgg_inductor.py

```
@@ -0,0 +1,146 @@
+    print("-" * 68)
+
+    baseline_ms = None
+    for label, build_fn in configs:
+        model = build_fn()
+        model.eval()
+
+        with torch.no_grad():
+            # One extra forward to initialise states and trigger Triton JIT +
+            # autotune before timing begins (avoids measuring compile cost).
+            model(x)
+            ms = cuda_time_ms(lambda: (functional.reset_net(model), model(x)))
```

Ruff (0.15.11)

[warning] Function definition does not bind loop variable `model`

(B023)